.github/workflows/python-package.yml

```
@@ -41,4 +41,4 @@ jobs:
         python3 -m coverage run --source=./RadClass/ -m pytest
         python3 -m coverage report
         python3 -m coverage html
-        python3 -m coveralls
+        python3 -m COVERALLS_REPO_TOKEN=${{ secrets.COVERALLS_REPO_TOKEN }} coveralls
```

The syntax is to set the ENV variable before running the executable.  In this case the executable is `python3`
```suggestion
        COVERALLS_REPO_TOKEN=${{ secrets.COVERALLS_REPO_TOKEN }} python3 -m  coveralls
```